### PR TITLE
Added privacy policy and updated splash screen

### PR DESCRIPTION
**Pull request by akashdabhi03**
This PR tracks issue #3

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <resources>
     <string name="device_admin_label">onebillion</string>
     <string name="device_admin_description">Admin Device Policies Description</string>
-    <string name="content_extraction" translatable="false">यह प्रक्रिया केवल एक बार की जाएगी।</string>
+    <string name="content_extraction" translatable="false">यह प्रक्रिया केवल एक बार की जाएगी। इसमें 10 मिनट लग सक्ते हैं।</string>
```
Comment (ankitk20) [inline app/src/main/res/values/strings.xml]: इसमें १० मिनट तक का समय लग सकता हैं।

Comment (mathewnik90) [inline app/src/main/res/values/strings.xml]: If we are adding hindi full stop, please keep it consistent

Comment (akashdabhi03) [reply app/src/main/res/values/strings.xml]: @ankitk20 @Shailin @harry6396 
Note this point

Comment (akashdabhi03) [reply app/src/main/res/values/strings.xml]: Updated

**Review by ankitk20 — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <resources>
     <string name="device_admin_label">onebillion</string>
     <string name="device_admin_description">Admin Device Policies Description</string>
```
Comment (Shailin) [inline app/src/main/res/values/strings.xml:3]: Is this required after removing device admin

Comment (akashdabhi03) [reply app/src/main/res/values/strings.xml:3]: I have just set the key to false to remove device admin permission and hence I have not removed the code.

**Review by Shailin — APPROVED** [review]

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**